thoth/storages/graph/postgres.py

```diff
@@ -6270,3 +6270,31 @@ def get_python_rule_all(
                 result.append(self._rule_to_dict(rule))
 
             return result
+
+    def get_python_package_version_solver_rules(
```

Comment:
suffix `_all()` should be added at the end based on convention: https://github.com/thoth-station/storages/tree/master/docs/conventions

Reply:
Adjusted in 9e09060 PTAL 👍🏻